bin/updateDoc

```diff
@@ -0,0 +1,317 @@
+        return implode(PHP_EOL, $docComment);
+    }
+
+    $methodHelp = '';
+    foreach ($methodHelpLines as $methodHelpLine) {
+        $methodHelpLine = preg_replace('/^.. (\w+)::/', '$1: ', $methodHelpLine);
+        $methodHelpLine = preg_replace('/:ref:`(\w+) <.*?>`/', '$1', $methodHelpLine);
+
+        if (preg_match('/^\s*..\s+[\w\-]+:/', $methodHelpLine)) {
+            continue;
+        }
+
+        if (preg_match('/^\s+\|\s/', $methodHelpLine)) {
+            $methodHelpLine = ltrim($methodHelpLine, ' |');
+        }
+
+        $methodHelp .= str_repeat(' ', $spacesCount) . '* ' . rtrim($methodHelpLine) . PHP_EOL;
+    }
+
+    $methodHelp = preg_replace('/(?:\s{' . $spacesCount . '}\*\s*\n)+/', str_repeat(' ', $spacesCount) . '*' . PHP_EOL, $methodHelp);
+    $methodHelp = rtrim($methodHelp);
+
+    $doc = '/**' . PHP_EOL;
+    $doc .= $methodHelp;
+    $doc .= $methodHelp !== '' ? PHP_EOL : '';
+    $doc .= implode(PHP_EOL, array_slice($docComment, $firstAnnotation));
+
+    return $doc;
+}
+
+
+function replaceInFile($file, $docComment, $newDocComment, $suffix)
+{
+	return file_put_contents(
+		$file,
+		preg_replace_callback(
+			sprintf('/(%s)([\w\s]+%s)/msi', preg_quote($docComment, '/'), preg_quote($suffix, '/s')),
+			function ($matches) use ($newDocComment) {
+				return $newDocComment . $matches[2];
+			},
+			file_get_contents($file)
+		)
+	);
+}
+
+function updateFile($file, $className, array $helps)
+{
+	$refectionClass = new \ReflectionClass($className);
+	foreach ($refectionClass->getMethods() as $method) {
+		if ($className != $method->getDeclaringClass()->getName()) {
+			continue;
+		}
+
+		if (strtolower($method->getName()) == 'isreferenceto') {
+			continue;
+		}
+
+		$newDocComment = replaceHelpDoc(
+			$method->getDocComment(),
+			5,
+			function() use ($helps, $method) {
+				return getMethodHelp($method, $helps);
+			}
+		);
+
+		replaceInFile($file, $method->getDocComment(), $newDocComment, 'function ' . $method->getName() . "(");
+	}
+
+	foreach ($refectionClass->getProperties() as $property) {
+		if ($className != $property->getDeclaringClass()->getName()) {
+			continue;
+		}
+
+		$newPropertyDocComment = replaceHelpDoc(
+			$property->getDocComment(),
+			5,
+			function() use ($helps, $property){
+				return getPropertyHelp($property, $helps);
+			}
+		);
+
+		replaceInFile($file, $property->getDocComment(), $newPropertyDocComment, 'public $' . $property->getName() . ";");
+	}
+
+	if (false !== $refectionClass->getDocComment()) {
+		$newClassDocComment = replaceHelpDoc(
+			$refectionClass->getDocComment(),
+			1,
+			function() use ($helps, $refectionClass){
+				return getClassHelp($refectionClass, $helps);
+			}
+		);
+		replaceInFile($file, $refectionClass->getDocComment(), $newClassDocComment, $refectionClass->getShortName());
+	}
+}
+
+passthru('rm -rf tmp && mkdir -p tmp && cd tmp && git clone git@github.com:atoum/atoum-documentation.git && cd atoum-documentation && ./builddoc text');
```

Comment:
can you use the `https` clone url as it will be clonable by anyone without any restriction.